website/docs/cloud-docs/workspaces/settings/deletion.mdx

```diff
@@ -47,9 +54,22 @@ To cancel a scheduled auto destroy run in Terraform Cloud:
 1. Under **Automatically destroy**, click **Edit** next to your scheduled run's details.
 1. Click **Remove**.
 
-You can schedule [notifications](/terraform/cloud-docs/workspaces/settings/notifications) to alert you 12 and 24 hours before an auto destroy run, and to report auto destroy run results.
+#### Destroy if Inactive
+
+You can configure Terraform Cloud to automatically destroy a workspace's infrastructure after a period of inactivity. Any runs that update Terraform state will push back the scheduled auto destroy time by the configured activity duration designated by the user.
+
```

Comment:
Could you also add an extra sentence here making it super clear that this setting _persists_ after the first auto-destroy run, unlike the other auto-destroy setting?

Reply:
Added this, let me know if it needs to any updates

```md
!> **Note:** As opposed to configuring an auto-destroy run for a specific date and time, this setting will _persist_ after queued auto-destroy runs.
```